pages/linux/cpupower.md

```diff
@@ -0,0 +1,25 @@
+# cpupower
+
+> Tools regarding CPU power and tuning options.
+> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.
+> More information: <https://manned.org/cpupower>.
+
+- List CPUs:
+
+`sudo cpupower --cpu cpulist`
```

Comment:
Does this command work for you as is? I get an `Error parsing cpu list`. From the man page it seems that `cpulist` would have to be replaced with something like `all`, but then it needs another subcommand like `info`.

Reply:
That command is _meant_ to work (going off what the manpages said), but it does seem to break a lot on many installs. I suggested `sudo cpupower -c all info` which indirectly serves the same purpose.
If it's breaking for everyone that's trying it, I'll just go and change it.